src/d3a/models/area/redis_dispatcher/area_event_dispatcher.py

```diff
@@ -0,0 +1,58 @@
+import json
+from random import random
+from d3a.events import AreaEvent
+from d3a.models.area.redis_dispatcher import RedisEventDispatcherBase
+
+
+class AreaRedisAreaEventDispatcher(RedisEventDispatcherBase):
```

Comment:
Umm, may be overly nit-picky, but wouldn't `RedisAreaEventDispatcher` be enough to know the purpose of this class.

Reply:
Judging from the fact that this resides in the area package then you are probably right, however @hannesdiedrich needs to agree here because I changed this after his comment. Personally both look good to my eyes, AreaRedisAreaEventDispatcher is more lengthy but still easily understandable.

Reply:
I have no idea what came over me when I made the comment. :-) 
There is no other place where an Area Event is Dispatched, right? 
So, please go ahead and rename it to `RedisAreaEventDispatcher`. Thanks!

Reply:
Fixed